piDIOComm.c

```diff
@@ -116,38 +116,32 @@ INT32U piDIOComm_Config(uint8_t i8uAddress, uint16_t i16uNumEntries, SEntryInfo
 
 INT32U piDIOComm_Init(INT8U i8uDevice_p)
 {
-	u8 addr = RevPiDevice_getDev(i8uDevice_p)->i8uAddress;
```

Comment:
I'd retain this line. That way you can use `addr` in the call to `pibridge_req_io()` and stay below 80 chars per line.